ext/openssl/ossl_pkcs7.c

```diff
@@ -878,6 +889,24 @@ ossl_pkcs7_to_pem(VALUE self)
     return str;
 }
 
+static VALUE
+ossl_pkcs7_finalize(VALUE self, VALUE data, VALUE flags)
+{
+    PKCS7 *pkcs7;
+    BIO *in;
+
+    GetPKCS7(self, pkcs7);
+
+    int flg = NIL_P(flags) ? 0 : NUM2INT(flags);
```

Comment:
Style: Until we drop support for Ruby 2.6, we have to avoid C99 features (`//` comments and mixed declarations and statements).